### ISSUE TITLE
Javadoc Check should log warning if current Javadoc comment has unclosed HTML tag and the Check doesn't support that

### ISSUE THREAD

**Issue by baratali** [checkstyle/checkstyle#3313]
After this issue is resolved https://github.com/checkstyle/checkstyle/issues/3311 we need to make Javadoc Checks log warning if current Javadoc comment has unclosed HTML tag and the Check doesn't support that.

**Comment by Gaurav-Punjabi** [comment]
@baratali [JavadocStyle](https://checkstyle.sourceforge.io/config_javadoc.html#JavadocStyle) already checks for unclosed HTML tags. Do you want any other violations regarding HTML content  ?

**Comment by baratali** [comment]
@Gaurav-Punjabi , purpose of this task is to print a warning if any Javadoc Check cannot check javadoc comment if there is an unclosed html tag (because AST tree breaks). So we would warn that a Check cannot do its work untill there is an unclosed tag.

Actually, this is already done, but in a different way. New property `violateExecutionOnNonTightHtml` was added in #3311. 
If its value is true, Check will print a warning.
See this part:
https://github.com/checkstyle/checkstyle/blob/164a755af951cf0fd459d70873e1c199210d9d8b/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java#L320

Example:
```java
/**
 * <p>test
 * @deprecated test
 * @author test
 */
class Input {}
```
```xml
<!DOCTYPE module PUBLIC
  "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
  "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
   <module name="AtclauseOrder">
     <property name="violateExecutionOnNonTightHtml" value="true"/>
   </module>
  </module>
</module>
```
```bash
$ java -jar checkstyle-8.33-SNAPSHOT-all.jar -c config.xml Input.java
Starting audit...
[ERROR] C:\Users\Baratali\git\test_checkstyle\Input.java:2: Unclosed HTML tag found: p [AtclauseOrder]
[ERROR] C:\Users\Baratali\git\test_checkstyle\Input.java:4: At-clauses have to appear in the order '[@author, @version, @param, @return, @throws, @exception, @see, @since, @serial, @serialField, @serialData, @deprecated]'. [AtclauseOrder]
Audit done.
Checkstyle ends with 2 errors.
```

----

As an improvement, I would suggest to print different message if `acceptJavadocWithNonTightHtml()=false` **AND** `violateExecutionOnNonTightHtml=true`:
`Cannot proceed - unclosed HTML tag found ...` to specify that Check refuses to work in this case.

However, I don't see any Checks having `acceptJavadocWithNonTightHtml=false` now, so probably it's not worth it :)

@romani, does it make sense? Otherwise we can close this issue
